### PR TITLE
Download get-kube.sh if it doesn't exist

### PR DESCRIPTION
**Pull request by ixdy**
It seems I broke soak test deployment. This should fix it.

cc @Random-Liu 

https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/kubernetes-soak-weekly-deploy-gce-cri/1

```
+ export KUBERNETES_RELEASE
+ export KUBERNETES_RELEASE_URL
++ dirname /dev/fd/63
+ KUBERNETES_SKIP_CONFIRM=y
+ KUBERNETES_SKIP_CREATE_CLUSTER=y
+ KUBERNETES_DOWNLOAD_TESTS=y
+ /dev/fd/get-kube.sh
/dev/fd/63: line 128: /dev/fd/get-kube.sh: No such file or directory
+ rc=1
```

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/927)

### CODE REVIEW

File: jenkins/e2e-image/e2e-runner.sh [hunk 1]
```diff
@@ -123,19 +123,28 @@ function set_release_vars_from_gke_cluster_version() {
 
 function call_get_kube() {
     [[ "${JENKINS_USE_GET_KUBE_SCRIPT:-}" =~ ^[yY]$ ]] || exit 2
+    local get_kube_sh="${WORKSPACE}/get-kube.sh"
+    if [[ ! -x "${get_kube_sh}" ]]; then
+        # If running outside docker (e.g. in soak tests) we may not have the
```
Comment (fejta) [inline jenkins/e2e-image/e2e-runner.sh]: Any thoughts on moving this inside docker?

File: jenkins/e2e-image/e2e-runner.sh [hunk 2]
```diff
@@ -123,19 +123,28 @@ function set_release_vars_from_gke_cluster_version() {
...
     if [[ ! -x kubernetes/cluster/get-kube-binaries.sh ]]; then
-      # If the get-kube-binaries.sh script doesn't exist, assume this is an older
-      # release without it, and thus the tests haven't been downloaded yet.
-      # We'll have to download and extract them ourselves instead.
-      echo "Grabbing test binaries since cluster/get-kube-binaries.sh does not exist."
-      local -r test_tarball=kubernetes-test.tar.gz
-      curl -L "${KUBERNETES_RELEASE_URL:-https://storage.googleapis.com/kubernetes-release/release}/${KUBERNETES_RELEASE}/${test_tarball}" -o "${test_tarball}"
-      md5sum "${test_tarball}"
-      tar -xzf "${test_tarball}"
+        # If the get-kube-binaries.sh script doesn't exist, assume this is an older
```
Comment (fejta) [inline jenkins/e2e-image/e2e-runner.sh]: When did we switch to 4 spaces?

Comment (ixdy) [inline jenkins/e2e-image/e2e-runner.sh]: this file is really inconsistent. I'll switch back.

Comment (ixdy) [inline jenkins/e2e-image/e2e-runner.sh]: this file is really inconsistent, apparently.

File: jenkins/e2e-image/e2e-runner.sh (hunk 1 above)
Comment (ixdy) [inline jenkins/e2e-image/e2e-runner.sh]: sure, let's try it! #928

**Comment by Random-Liu** [comment]
@ixdy Thanks for fixing this!

**Comment by ixdy** [comment]
@Random-Liu sorry for breaking it in the first place! :)

**Pull request merged**